src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -18,59 +18,6 @@ if not http then
     return
 end
 
---- Attempts to guess the pastebin ID from the given code or URL
-local function extractId(paste)
-    local patterns = {
-        "^([%a%d]+)$",
-        "^https?://pastebin.com/([%a%d]+)$",
-        "^pastebin.com/([%a%d]+)$",
-        "^https?://pastebin.com/raw/([%a%d]+)$",
-        "^pastebin.com/raw/([%a%d]+)$",
-    }
-
-    for i = 1, #patterns do
-        local code = paste:match( patterns[i] )
-        if code then return code end
-    end
-
-    return nil
-end
-
-local function get(url)
-    local paste = extractId( url )
-    if not paste then
-        io.stderr:write( "Invalid pastebin code.\n" )
-        io.write( "The code is the ID at the end of the pastebin.com URL.\n" )
```

Comment:
I assume this message is no longer displayed? It's a bit of a pain, as that additional information is no longer present.